src/components/Attachments/AttachmentCarousel/index.native.js

```diff
@@ -87,6 +87,22 @@ function AttachmentCarousel({report, reportActions, parentReportActions, source,
         [autoHideArrows, page, updatePage],
     );
 
+    /**
+     * Toggles the arrows visibility
+     * @param {Boolean} showArrows if showArrows is passed, it will set the visibility to the passed value
+     */
+    const toggleArrows = useCallback(
+        (showArrows) => {
+            if (showArrows) {
+                setShouldShowArrows(showArrows);
+                return;
+            }
+
+            setShouldShowArrows(!shouldShowArrows);
+        },
+        [setShouldShowArrows, shouldShowArrows],
+    );
```

Comment:
Passing false should set the state value to false.

We should toggle only if the passed. value is undefined

Reply:
you're right, my mistake.. sorry it's late, if there are any other comments i'm gonna address them tmrw morning :)